cfgov/paying_for_college/models/disclosures.py

```diff
@@ -0,0 +1,771 @@
+            return region
+    return ''
+
+
+def make_divisible_by_6(value):
+    """Makes sure a value, such as program_length, is divisible by 6"""
+    if not value or value % 6 == 0:
+        return value
+    else:
+        return value + (6 - (value % 6))
+
+
+@python_2_unicode_compatible
+class ConstantRate(models.Model):
+    """Rate values that generally only change annually"""
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=255,
+                            blank=True,
+                            help_text="VARIABLE NAME FOR JS")
+    value = models.DecimalField(max_digits=6, decimal_places=5)
+    note = models.TextField(blank=True)
+    updated = models.DateField(auto_now=True)
+
+    def __str__(self):
+        return "{} ({}), updated {}".format(self.name, self.slug, self.updated)
+
+    class Meta:
+        ordering = ['slug']
+
+
+@python_2_unicode_compatible
+class ConstantCap(models.Model):
+    """Cap values that generally only change annually"""
+    name = models.CharField(max_length=255)
+    slug = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="VARIABLE NAME FOR JS")
+    value = models.IntegerField()
+    note = models.TextField(blank=True)
+    updated = models.DateField(auto_now=True)
+
+    def __str__(self):
+        return "{} ({}), updated {}".format(self.name, self.slug, self.updated)
+
+    class Meta:
+        ordering = ['slug']
+
+
+# original data_json fields:
+# ALIAS -- not needed, DELETE
+# AVGMONTHLYPAY
+# AVGSTULOANDEBT
+# AVGSTULOANDEBTRANK -- not needed, DELETE
+# BADALIAS -- not needed, DELETE
+# BAH 1356 -- not needed, DELETE
+# BOOKS
+# CITY (now school.city)
+# CONTROL (now school.control)
+# DEFAULTRATE
+# GRADRATE -- now school.grad_rate
+# GRADRATERANK -- not needed, DELETE
+# INDICATORGROUP
+# KBYOSS (now school.kbyoss) -- not needed, DELETE
+# MEDIANDEBTCOMPLETER # new in 2015
+# NETPRICE110K -- not needed, DELETE
+# NETPRICE3OK -- not needed, DELETE
+# NETPRICE48K -- not needed, DELETE
+# NETPRICE75K -- not needed, DELETE
+# NETPRICEGENERAL -- not needed, DELETE
+# NETPRICEOK -- not needed, DELETE
+# OFFERAA
+# OFFERBA
+# OFFERGRAD
+# ONCAMPUSAVAIL
+# ONLINE (now school.online)
+# OTHEROFFCAMPUS
+# OTHERONCAMPUS
+# OTHERWFAMILY
+# RETENTRATE -- not needed, DELETE
+# RETENTRATELT4 # new in 2015 -- not needed, DELETE
+# REPAY3YR # new in 2015
+# ROOMBRDOFFCAMPUS
+# ROOMBRDONCAMPUS
+# SCHOOL (now school.primary_alias)
+# SCHOOL_ID (now school.pk)
+# STATE (now school.state)
+# TUITIONGRADINDIS
+# TUITIONGRADINS
+# TUITIONGRADOSS
+# TUITIONUNDERINDIS
+# TUITIONUNDERINS
+# TUITIONUNDEROSS
+# ZIP (now school.zip5)
+
+
+@python_2_unicode_compatible
+class Contact(models.Model):
+    """school endpoint or email to which we send confirmations"""
+    contacts = models.TextField(
+        help_text="COMMA-SEPARATED LIST OF EMAILS",
+        blank=True)
+    endpoint = models.TextField(blank=True)
```

Comment:
See comment above about removing the `max_length` restriction on the `endpoint` and `name` fields here.

Reply:
Model and migration restored to varchar.